data/src/main/java/io/gnosis/data/models/transaction/TransferInfo.kt

```diff
@@ -1,33 +1,50 @@
 package io.gnosis.data.models.transaction
 
+import com.squareup.moshi.Json
 import io.gnosis.data.repositories.TokenRepository
 import pm.gnosis.model.Solidity
 import java.math.BigInteger
 
 enum class TransferType {
-    ERC20, ERC721, ETHER
+    ERC20,
```

Comment:
Should we have a \@Json annotation for the enum values too?

Reply:
These types are defined by client gw. So I would just reuse them. Defining annotations for these enums will make it less readable and we will have 2 sets of strings to maintain: one here and another one in MoshiBuilder

Reply:
Then why do we annotate anything? All types and names are defined by the client gateway. 

As I understand it we annotate, so:
1. We can refactor everything without breaking the parsing 
2. To use moshi codegen instead of reflection

Reply:
convincing argument